### ISSUE TITLE
Unexpected communication loss with the proto-type ARX board.

### ISSUE THREAD

**Issue by rh-codebase** [ovro-lwa/lwa-issues#28]
After reverting to previous stable code, the communication was still absent.	I finally had Mark unplug	the USB	to 485 dongle and reattach. The device switched: /dev/ttyUSB0 -> /dev/ttyUSB1( which may just be a linux timeout issue) and this fixed the communication.                                                                                                                                                                            
I don't know if this is a dongle problem, a Linux kernel issue or ARX issue. Does unplugging the dongle affect the ARX board? If this happens again, a hard reset of the ARX board\ can be done and the outcome noted.

**Comment by rh-codebase** [comment]
The device /dev/ttyUSB0 was still present so the OS still had it allocated. I restarted the app. a number of times, and it successfully connected. dmesg showed the last time the device was detected and correctly showed the device disconnecting and reconnecting yesterday(times are UT): 

'[Wed May 26 00:18:44 2021] usb 3-1: Manufacturer: Silicon Labs
[Wed May 26 00:18:44 2021] usb 3-1: SerialNumber: 0001
[Wed May 26 00:18:44 2021] usb 3-1: cp210x converter now attached to ttyUSB0
[Thu Jun  3 00:39:14 2021] usb 3-1: USB disconnect, device number 4
[Thu Jun  3 00:39:14 2021] cp210x ttyUSB0: failed set request 0x12 status: -19
[Thu Jun  3 00:39:14 2021] cp210x ttyUSB0: failed set request 0x0 status: -19
[Thu Jun  3 00:39:14 2021] cp210x ttyUSB0: cp210x converter now disconnected from ttyUSB0
[Thu Jun  3 00:39:20 2021] usb 3-2: new full-speed USB device number 5 using xhci_hcd
[Thu Jun  3 00:39:21 2021] usb 3-2: New USB device found, idVendor=10c4, idProduct=ea60
[Thu Jun  3 00:39:21 2021] usb 3-2: New USB device strings: Mfr=1, Product=2, SerialNumber=3
[Thu Jun  3 00:39:21 2021] usb 3-2: Product: CP2102 USB to UART Bridge Controller
[Thu Jun  3 00:39:21 2021] usb 3-2: Manufacturer: Silicon Labs
[Thu Jun  3 00:39:21 2021] usb 3-2: SerialNumber: 0001
[Thu Jun  3 00:39:21 2021] usb 3-2: cp210x converter now attached to ttyUSB1
'

When t\he USB device was reconnected and the OS switched to /dev/ttyUSB1, it removed /dev/ttyUSB0. The application then reported an error that it could no longer connect to /dev/ttyUSB0 as it should have. Changing the application(service) code to use /dev/ttyUSB1 resulted in a successful connection and response from the ARX board.  
Nothing on the Linux side appears out of the ordinary.

**Comment by rh-codebase** [comment]
I've managed to lose communication to the ARX board again while testing the OWTE command via get_1wire_temp() in the python ARX client. I noticed, after executing the get_1wire_temp() command, get_chan_cfg() which runs GETC would fail. Running reset(RSET), followed by get_chan_cfg() worked. I then used the raw() function to reproduce:
`# get ARX object, ma
ma.raw(2, 'OWTE', 0.8)
ma.raw(2, 'GETC2')`
Afterwards, all communication to the device returned a timeout error. I increased the serial port timeout to 900ms and still receiving the timeout on basic commands. No difference when restarting the service.

Will file a separate issue regarding the bug related to OWTE.
